import-service/src/importFileParser.js

```diff
@@ -0,0 +1,44 @@
+const AWS = require("aws-sdk");
+const util = require("util");
+const stream = require("stream");
+const csv = require("csv-parser");
+
+const {BUCKET, REGION} = process.env;
+
+const finished = util.promisify(stream.finished);
+
+export const importFileParser = () => async (event) => {
+    console.log('start importFileParser')
+    const s3 = new AWS.S3({region: REGION});
+
+
+    for (const record of event.Records) {
+        const results = [];
+        const s3Stream = s3.getObject({
```

Comment:
Should do asynchronously, All can break if you put a big file with several chunks.
if you want covered this option "(for JS only) - async/await is used in lambda functions"
Can you rewrite?